NSS-Therapeutic-Raccoons/BangazonSite
# finished inital setup

## Link to Ticket
Closes Ticket # [34](https://github.com/NSS-Therapeutic-Raccoons/BangazonSite/issues/34)
* finished initial setup
* fixed routes for navbar
* scaffolded neccessary controllers for comitted tickets via team agreement
* edited default home screen view to not contain the standard content in preparation for coming tickets
* Drew image of Andy to look over up and give us confidence

 ## Description of Proposed Changes
- Created setup files, including initial data seeding, migration creation, db instantiation, and edited views and added Controllers for products and producttypes 
 
## Steps to Test
pull down, follow readme steps
 ```sh
git fetch --all
git checkout initial-setup
```
check for the goods

 ## Impacted Areas in Application
 List general components of the application that this PR will affect:
 * Database
 * Controllers/ProductsController
 * Controllers/ProductTypesController
 * Data/the files here that has the stuff in it

 ## Mentions @username
 @figamus
@ricky-bruner
@markhale94
@taylorlgulley
@askingalot
 ## Definition of Done
 1. The project must be fully documented. This includes the following:
	* Comments for classes and methods added
	* Comments for models, controllers, and views added
2. The testing steps must completely work per ticket requirements.
3. Andy must smile approvingly and smirk with satisfaction following the smile.
